builtin/providers/aws/resource_aws_dms_endpoint.go

```diff
@@ -283,11 +282,6 @@ func resourceAwsDmsEndpointDelete(d *schema.ResourceData, meta interface{}) erro
 		return err
 	}
 
-	waitErr := waitForEndpointDelete(conn, d.Get("endpoint_id").(string), 30, 20)
-	if waitErr != nil {
-		return waitErr
-	}
-
```

Comment:
Is it OK to no longer wait for deletion here? Waiting for Delete is usually a lesser concern than say waiting for creation, but it's still caused dependency issues in tear-down before